plotting/metallicity_distribution.py

```diff
@@ -49,7 +56,10 @@
 
 
 ax.legend(loc="upper right")
-ax.set_xlabel("Smoothed Metal Mass Fractions $Z$ []")
+if "COLIBRE" in ptype:
+        ax.set_xlabel("Metal Mass Fractions $Z$ []")
```

Comment:
Here then we can do `ax.set_xlabel(f"{'Smoothed ' if smoothed else ''}Metal Mass Fractions $Z$ []")`